pkg/nsx/services/common/store.go

```diff
@@ -0,0 +1,87 @@
+package common
+
+import (
+	error2 "errors"
+
+	"github.com/vmware/vsphere-automation-sdk-go/lib/vapi/std/errors"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+	"k8s.io/client-go/tools/cache"
+
+	util2 "github.com/vmware-tanzu/nsx-operator/pkg/nsx/util"
```

Comment:
Please don't use `error2` and `util2` as the names...

Reply:
OK, renamed all error2 util2 in repo.